### ISSUE TITLE
Not working?

### ISSUE THREAD

**Issue by scruffynerf** [rezreal/biogroove#1]
Neat idea, but the github.io site doesn't seem to work...

**Comment by rezreal** [comment]
yeah unfortunately lost motivation at some point so feel free to pick it up :D
